.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
+[target.x86_64-unknown-linux-gnu]
+linker = "x86_64-linux-gnu-gcc"
```

Comment:
Can we add this file only to the docker container? This breaks building the project on the host machine when the gnu-gcc compiler isn't installed.

```
error: linker `x86_64-linux-gnu-gcc` not found
  |
  = note: No such file or directory (os error 2)
```